modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -190,6 +190,15 @@ CallParams read<CallParams>(const cv::FileNode& fn) {
     return CallParams{std::move(name), static_cast<size_t>(call_every_nth)};
 }
 
+template <typename V>
+std::map<std::string, V> readMap(const cv::FileNode& fn) {
+    std::map<std::string, V> map;
+    for (auto item : fn) {
```

Comment:
`&&`

Reply:
I guess `FileNode` is copy-free, but I don't mind this change

Reply:
Done